src/Button.js

```diff
@@ -1,11 +1,92 @@
-import React from 'react'
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import F from 'futil'
+import styled from '@emotion/styled'
+import { Subtitle, Text, Icon, Flex } from './'
+import theme from './theme'
 
-let Button = ({ isActive, primary, as: As = 'button', className, ...x }) => (
+let getColor = F.aliasIn(theme.colors)
+
+let buttonStyle = (baseColor, hoverColor, activeColor, textColor) => ({
+  backgroundColor: getColor(baseColor),
+  ':hover': { backgroundColor: getColor(hoverColor) },
+  ':active': { backgroundColor: getColor(activeColor) },
+  '& *': { color: getColor(textColor) },
+})
+
+let ButtonText = ({ compact = false, large = false, ...props }) =>
```

Comment:
This implementation is a bit confusing - we might just want an explicit size prop since this is internal

Reply:
wow yeah that would be way better 🤦‍♀